sdk/log/logger.go

```diff
@@ -96,6 +123,9 @@ func (l *logger) newRecord(ctx context.Context, r log.Record) Record {
 		attributeCountLimit:       l.provider.attributeCountLimit,
 		allowDupKeys:              l.provider.allowDupKeys,
 	}
+	if l.selfObservabilityEnabled {
+		l.logCreatedMetric.Add(ctx, 1)
```

Comment:
Is the `ctx` here correct? cc @pellared

Reply:
I think it is correct as well.  It is funny that you can end up with self-obs metrics with exemplars that point to random user spans, but that seems like it is WAI to me.  It would answer questions like "What the heck is writing so many log records?"